app/models/main.py

```diff
@@ -1658,6 +1669,11 @@ class BriefResponse(db.Model):
 
     award_details = db.Column(JSON, nullable=True, default={})
 
+    __table_args__ = (
+        # this may appear tautological, but is required to be able to make a compound foreign key to these together
+        db.UniqueConstraint(id, brief_id, name="uq_brief_responses_id_brief_id"),
```

Comment:
I'm a bit confused by what `brief_id` is here if it isn't the same as `id`?

Reply:
The reason it's tautological is that if `id` is unique (which we know it is, it's a primary key) then *of course* `(id, brief_id)` is. But postgres (at time of writing) don't care. It wants a unique index on the column combination it's been asked to make a foreign key to. Known postgres ~~~bug~~~ imperfection, can't find a reference link to right now.

Reply:
(was that comment not clear enough?)